### DISCUSSION TITLE
Hello everyone, I would like to ask you about the local compilation of code-server

### DISCUSSION THREAD

**Discussion by Pythoneryeah** [coder/code-server#6379] [Q&A]
background note：
1. I plan to follow these steps to compile and start code-server
![image](https://github.com/coder/code-server/assets/37642828/6d01e35a-cb3f-459f-9bfc-4dc009bde273)
2. Due to network issues, I chose to manually download vscode to the lib/vscode submodule when "git submodule update -- init-Clone vscode submodule".
3. After vscode was downloaded to lib/vscode, I executed "git submodule update --init" and no error was reported
4. If yarn is executed, the message "Yarn" is also displayed
5. However, when yarn watch is executed, the following error message is displayed:

![image](https://github.com/coder/code-server/assets/37642828/57bc7437-07fa-4166-a2b5-2044f0cd747a)

**Tried a lot of ways, can not solve, may I ask what is the problem?**

**Comment by code-asher** [comment]
Hmm not sure.  What did you do to download it manually?  By 
downloading the zip off GitHub or something like that?  Let me 
know and I will try it to see if I get the same issue.
